pkg/util/util_supported.go

```diff
@@ -19,6 +19,12 @@ var (
 	rootlessRuntimeDir     string
 )
 
+// isWriteableOnlyByOwner checks that the specified permission mask allows write
+// access only to the owner.
+func isWriteableOnlyByOwner(perm os.FileMode) bool {
```

Comment:
I find the function name misleading: yes, it checks `WritableOnlyByOwner` but also checks that owner has `rx`. Unfortunately I don't know the context for checking that so I don't know when if or how user perms would not be rwx, or why there's a need to check. 

As best I can tell from the original commit 4cc4c7137d29111fe6ccca526bc11d435fa20dd7, I believe the intention behind this check was more "does owner have rwx" than "is it locked down". If so, a name like `ownerHasFullAccess()` might be more descriptive - but I really don't know. @giuseppe this was your code from the beginning, though: what was your intention then (July 2018)? Can you (or anyone) find a less misleading name for this check?

Reply:
Oh wow, github didn't linkify the commit hash. Try this instead: #1048

Reply:
Wow, another failure to linkify. This? https://github.com/containers/podman/pull/1048